README.md

```diff
@@ -12,7 +12,14 @@ Add this Chart repo to Helm, and install:
 
 ```console
 helm repo add gremlin https://helm.gremlin.com/
-helm install gremlin/gremlin --set gremlin.teamID=YOUR-TEAM-ID
```

Comment:
No steps in this README about externally managing a secret, as such I decided it best to replace this with an example that uses managed secrets